less/pages/single-form.less

```diff
@@ -1,3 +1,11 @@
 .payment-section {
   padding-top: 25px;
 }
+
+#section-heading {
+  margin-bottom:14px;
```

Comment:
I think we can accomplish the same thing by removing this line: https://github.com/mozilla/donate.mozilla.org/blob/971cb3793f90b94c9a3b73d8b95f8dfee7e8938d/less/shared.less#L472